esmvalcore/dataset.py

```diff
@@ -96,6 +97,22 @@ class Dataset:
         Facets describing the dataset.
     """
 
+    _SUMMARY_FACETS = (
```

Comment:
```suggestion
    _summary_facets = (
```
Maybe use lowercase here to be consistent with other places where we have class attributes?

Reply:
PEP8 [recommends](https://peps.python.org/pep-0008/#constants) to write constants in all-capital letter. We also have examples in our code where class attributes are capitals, so I'd like to leave it like this.